sympy/core/expr.py

```diff
@@ -2845,6 +2845,46 @@ def diff(self, *symbols, **assumptions):
         assumptions.setdefault("evaluate", True)
         return Derivative(self, *new_symbols, **assumptions)
 
+    def differenceDelta(self, n=None, step=1):
```

Comment:
Common python convention is to use snake_case instead of camelCase. So this should be `difference_delta`.